client/src/views/page-components/calendar-page/Calendar/Calendar.scss

```
@@ -0,0 +1,77 @@
+@mixin tableCellFontStyleMixin {
+    font-family: Raleway;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 12px;
+    color: var(--concept-color);
+    
+}
+
+.calendar__table{
+    width:100%;
+    height:100%;
+    border-collapse: separate;
+    border-spacing: 15px 8px;
+
+    thead{
+        display:table-header-group;
+
+        tr{
+            display:table-row;
+
+            td{
+                @include tableCellFontStyleMixin;
+
+                display:table-cell;
+                height:100px;
+                width:130px;
+
+                text-align:center;
+                opacity:0.5;
+                
+                &:first-child{
+                    color: #EF5D5D;
+                }
+            }
+        }
+    }
+    
+    tbody{
+        display:table-row-group;
+
+        tr{
+            display:table-row;
+            vertical-align: inherit;
+
+            td{
+                @include tableCellFontStyleMixin;
+                display:table-cell;
+
+                width:130px;
+                height:120px;
+                background-color:#fff;
+                border-radius:10px;
+                padding: 16px;
+
+
+                transition:all .3s ease;
```

이 부분 filter로 바꿔주시는게 더 좋을듯합니다!
```suggestion
                transition:filter .3s ease;
```

---

넵 수정했습니다~